src/components/api.js

```diff
@@ -2,8 +2,8 @@ import axios from "axios";
 
 const api = axios.create({ baseURL: "https://robs-nc-news.onrender.com" });
 
-const getAllArticles = () => {
-  return api.get("/api/articles").then((response) => {
+const getAllArticles = (query = "") => {
```

Comment:
I really like that you've extended the use of your existing`getArticles` function rather than make a separate function just to get the articles with a certain topic.
Your current implementation is great (and you don't need to change it) but FYI you could also use axios' params functionality to add queries to a request robustly https://notes.northcoders.com/courses/js-front-end/axios#axios-params